Test case test_pvc_delete_and_verify_size_is_returned_to_backend_pool failing in arbiter

tests/manage/pv_services/test_pvc_delete_verify_size_is_returned_to_backendpool.py::TestPVCDeleteAndVerifySizeIsReturnedToBackendPool::test_pvc_delete_and_verify_size_is_returned_to_backend_pool


The test case given above is filing in arbiter cluster.

```
# Write 6Gb
        pod.run_io_and_verify_mount_point(pod_obj, bs="10M", count="600")
        exp_size = used_before_io + (6 * replica_size)
>       used_after_io = helpers.fetch_used_size(cbp_name, exp_size)

tests/manage/pv_services/test_pvc_delete_verify_size_is_returned_to_backendpool.py:87: 
........
........
# Convert size to GB
        used_in_gb = float(format(size_bytes / constants.GB, ".4f"))
        if exp_val and abs(exp_val - used_in_gb) > 1.5:
>           raise UnexpectedBehaviour(
                f"Actual {used_in_gb} and expected size {exp_val} not "
                f"matching. Retrying"
            )
E           ocs_ci.ocs.exceptions.UnexpectedBehaviour: Actual 33.5199 and expected size 27.830399999999997 not matching. Retrying
```

The failure is in the used size verification after writing the initial set of data.

--- a/tests/manage/pv_services/test_pvc_delete_verify_size_is_returned_to_backendpool.py
+++ b/tests/manage/pv_services/test_pvc_delete_verify_size_is_returned_to_backendpool.py
@@ -65,8 +65,10 @@ class TestPVCDeleteAndVerifySizeIsReturnedToBackendPool(ManageTest):
 
         cbp_name = helpers.default_ceph_block_pool()
 
-        cbp_data = helpers.get_pool_cr(cbp_name)
-        replica_size = int(cbp_data["spec"]["replicated"]["size"])
+        tools_pod = pod.get_ceph_tools_pod()
+        cmd = f"ceph osd pool get {cbp_name} size"
+        size_info = tools_pod.exec_ceph_cmd(ceph_cmd=cmd)
+        replica_size = size_info("size")
 
         pvc_obj = pvc_factory(
             interface=constants.CEPHBLOCKPOOL, size=10, status=constants.STATUS_BOUND

--- a/tests/manage/pv_services/test_pvc_delete_verify_size_is_returned_to_backendpool.py
+++ b/tests/manage/pv_services/test_pvc_delete_verify_size_is_returned_to_backendpool.py
@@ -68,7 +68,7 @@ class TestPVCDeleteAndVerifySizeIsReturnedToBackendPool(ManageTest):
         tools_pod = pod.get_ceph_tools_pod()
         cmd = f"ceph osd pool get {cbp_name} size"
         size_info = tools_pod.exec_ceph_cmd(ceph_cmd=cmd)
-        replica_size = size_info("size")
+        replica_size = int(size_info["size"])
 
         pvc_obj = pvc_factory(
             interface=constants.CEPHBLOCKPOOL, size=10, status=constants.STATUS_BOUND

--- a/tests/manage/pv_services/test_pvc_delete_verify_size_is_returned_to_backendpool.py
+++ b/tests/manage/pv_services/test_pvc_delete_verify_size_is_returned_to_backendpool.py
@@ -65,8 +65,8 @@ class TestPVCDeleteAndVerifySizeIsReturnedToBackendPool(ManageTest):
 
         cbp_name = helpers.default_ceph_block_pool()
 
-        # TODO: Get exact value of replica size
-        replica_size = 3
+        cbp_data = helpers.get_pool_cr(cbp_name)
+        replica_size = int(cbp_data["spec"]["replicated"]["size"])
 
         pvc_obj = pvc_factory(
             interface=constants.CEPHBLOCKPOOL, size=10, status=constants.STATUS_BOUND

